CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/nCoVToolkit/nCoVSteppableBase.py

```diff
@@ -1,19 +1,472 @@
-# This is a general library of CompuCell3D steppable classes for the shared coronavirus modeling and simulation project
-# hosted by the Biocomplexity Institute at Indiana University
+"""
+This module defines the CompuCell3D base steppable class for the framework.
+
+Written by T.J. Sego, Ph.D.
+"""
+
+from collections import namedtuple
+from typing import *
 
 from cc3d.core.PySteppables import *
+from cc3d.cpp import CompuCell
+
+ode_manager_key = 'std_ode_manager_steppable'
+_registry_key = 'nCoVSteppableBase_registry'
+
+
+class NoODEManagerException(Exception):
+    pass
 
 
 class nCoVSteppableBase(SteppableBasePy):
```

Comment:
I still think this Steppable should include the register_cells methods, since that should be basic to all Steppables.

Reply:
Let's discuss this for the next minor version.